build.gradle

```diff
@@ -26,8 +26,8 @@ buildscript {
     classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.4'
     classpath 'org.jfrog.buildinfo:build-info-extractor-gradle:4.0.0'
     classpath 'com.android.tools.build:gradle:1.5.0'
-    classpath 'com.redhat.ceylon.gradle:android:0.0.1-SNAPSHOT'
-//    classpath "com.athaydes.gradle.ceylon:ceylon-gradle-plugin:1.1"
+//    classpath 'com.redhat.ceylon.gradle:android:0.0.1-SNAPSHOT'
```

Comment:
@bjansen how can the Gradle plugin support incremental compilation? If you mean incrementally build Gradle modules (ie. only build a module if it changes or some of its dependencies change), then this is already supported. If you mean incrementally analyse which ceylon files need to be re-compiled, then that's not possible unless Ceylon itself supports that, as far as I can tell.

Do you have any idea about how to get this done?